node-swc/src/index.ts

```diff
@@ -12,7 +13,10 @@ export * from "./types";
 import { BundleInput, compileBundleOptions } from "./spack";
 import { loadBinding } from "@node-rs/helper";
 
-const bindings = loadBinding(__dirname, "swc", "@swc/core")
+// Allow overrides to the location of the .node binding file
+const bindingsOverride = process.env["SWC_BINDING"];
```

Comment:
@alexeagle actually thinking this bit more - how much deatil we should expose?

currently it looks like it only cares about the path to the pkg, rest relies on what `loadBinding` assumes around pkg structure (https://github.com/napi-rs/node-rs/blob/main/packages/helper/src/loader.ts#L10)

But if we want full control, say bazel project have a pkg build to swc binary itself and want to point to path to those specific binary 1. will it follow same pkg structure `loadBinding` assumes? 2. will it follow same file name structures to the binary?

maybe something like `SWC_BINARY_PATH` which explicitly requires to absolute / relative path `includes binary file name`, then skips `loadBinding` for those cases would be better?

Reply:
Sure, makes sense, the napi-rs helper isn't helping for this use case. Changed as you suggested.